docs/body.md

```diff
@@ -748,39 +615,336 @@ when this happens.
+                           map_from:
+                               name: "network\\.(incoming|outgoing)\\.(bytes|packets)"
+                               unit: "(B|packet)"
+                        target:
+                            map_to:
+                                name: "network.\\1.\\2.rate"
+                                unit: "\\1/s"
+                            type: "gauge"
+              publishers:
+                  - https://10.0.0.10:8342/ceilometer/v1/samples/myopenstack?verify_ssl=False
+                  - https://10.0.0.12:8342/ceilometer/v1/samples/myopenstack?verify_ssl=False
+                  - <some_other_publisher>
+
+* Make sure you restart ceilometer-notification service whenever you change
+  `pipeline.yaml` or `event_pipeline.yaml`.
+
+#### RHOSP Overcloud Configuration from Undercloud (Director)
+
+*   When configuring Overcloud using templates, you need to add the
+    following to your environment template:
+
+            ManagePipeline: true
+            PipelinePublishers:
+              - https://<zenoss_ip>:8342/ceilometer/v1/samples/<zenoss_devicename>?verify_ssl=False
+              - <other_pipeline_publishers>
+            ManageEventPipeline: true
+            EventPipelinePublishers:
+              - https://<zenoss_ip>:8342/ceilometer/v1/events/<zenoss_devicename>?verify_ssl=False
+              - <other_event_pipeline_publishers>
+
+    For example:
+
+            ManagePipeline: true
+            PipelinePublishers:
+              - https://10.0.0.10:8342/ceilometer/v1/samples/myopenstack?verify_ssl=False
+              - https://10.0.0.12:8342/ceilometer/v1/samples/myopenstack?verify_ssl=False
+            ManageEventPipeline: true
+            EventPipelinePublishers:
+              - https://10.0.0.10:8342/ceilometer/v1/events/myopenstack?verify_ssl=False
+              - https://10.0.0.12:8342/ceilometer/v1/events/myopenstack?verify_ssl=False
+
+*   This template must be ***rendered*** into your templates before you initiate a deployment from
+    Undercloud.<br>
+    For more information on RHOSP template management, see RedHat's
+    [Including Environment Files in Overcloud Creation](https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html-single/advanced_overcloud_customization/index#sect-Including_Environment_Files_in_Overcloud_Creation).
+
+*   After deployment is complete, go to the undercloud, SSH into controller and go
+    to `/etc/ceilometer` in ceilometer_agent_notification container to check if
+    `pipeline.yaml` and `event_pipeline.yaml` file is updated:
+
+            ssh heat-admin@<controller ip>
+            sudo su -
+            sudo docker exec --user 0 -it ceilometer_agent_notification /bin/bash
+            # Ensure your configuration had the right publishers
+
+#### Notes:
+
+*   Events exposed by Ceilometer are listed in zOpenStackProcessEventTypes.
+    By default, this is set to the single *compute.instance.create.error* .
```

Comment:
I think we need to explain the change to how events are handled better.  Might need its own section.  I can write it if you'd like